djangocms_alias/admin.py

```diff
@@ -80,11 +85,20 @@ def delete_model(self, request, obj):
 @admin.register(AliasContent)
 class AliasContentAdmin(admin.ModelAdmin):
     form = AliasContentForm
+    list_filter = (LanguageFilter,)
 
     def save_model(self, request, obj, form, change):
         super().save_model(request, obj, form, change)
-        emit_content_change([obj], sender=self.model)
+
+        # Only emit content changes if versioning is not installed
+        # Versioning emits it's own signals for changes
+        if not is_versioning_enabled():
+            emit_content_change([obj], sender=self.model)
 
     def delete_model(self, request, obj):
         super().delete_model(request, obj)
-        emit_content_delete([obj], sender=self.model)
+
+        # Only emit content changes if versioning is not installed
```

Comment:
i think the comments are not really necessary as the method name / check are self-explanatory

Reply:
This was an issue and actually broke the package, I left a comment so that anyone in the future had additional information to know that a signal is still fired by versioning without this signal being triggered. Anyone in the future may take this exclusion as the signal never actually being fired when it is, just by versioning. Would you have know that without this comment? If so then it could be redundant if it's repeated and I've missed that.

Let me know if that changes your opinion or not. Ta.

Reply:
then the description above should be the comment :)